src/test/java/net/snowflake/ingest/streaming/internal/datatypes/IcebergNumericTypesIT.java

```diff
@@ -26,6 +34,8 @@ public static Object[][] parameters() {
   @Parameterized.Parameter(1)
   public static Constants.IcebergSerializationPolicy icebergSerializationPolicy;
 
+  static final Random generator = new Random(0x5EED);
```

Comment:
A fixed seed will imply the exact set of values being tested everytime. Why not have a randomly generated seed and plug that in here.
The reason to explicitly generate a seed and plug it in is - we can now log this seed / print it out in failure messages, and accurately reproduce any failures locally.

If you just do new Random(), reproducing a one-off error will be next to impossible.

Reply:
Switched to random seed.